bin/dbloader.py

```diff
@@ -152,11 +151,11 @@ def run_as_daemon(loader, interval):
                             action='version',
                             version=ver)
 
-    # Using the vars function to output a dict-like view rather than Namespace object.
-    options = vars(arg_parser.parse_args())
+    # Parsing arguments into an argparse.Namespace object for structured access.
+    options: Namespace = arg_parser.parse_args()
```

Comment:
What's the benefit of using a type hint here? How necessary is it? It breaks the old Travis CI test (though that'll be going eventually).

Reply:
There isn't - the comment above shows that the object is a Namespace object. I have removed the unnecessary bits.